test/handlers/generic/executeProposalV1.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright 2020 ChainSafe Systems
+ * SPDX-License-Identifier: LGPL-3.0-only
+ */
+
+const TruffleAssert = require('truffle-assertions');
+const Ethers = require('ethers');
+const Helpers = require('../../helpers');
+
+const CentrifugeAssetContract = artifacts.require("CentrifugeAsset");
+const GenericHandlerContract = artifacts.require("GenericHandlerV1");
+
+contract('GenericHandlerV1 - [Execute Proposal]', async (accounts) => {
+    const originDomainID = 1;
+    const destinationDomainID = 2;
+    const expectedDepositNonce = 1;
+
+    const depositorAddress = accounts[1];
+    const relayer1Address = accounts[2];
+    const relayer2Address = accounts[3];
+
+    const feeData = '0x';
+    const destinationMaxFee = 2000000;
+    const hashOfCentrifugeAsset = Ethers.utils.keccak256('0xc0ffee');
+    console.log("hashOfCentrifugeAsset",hashOfCentrifugeAsset)
+
+    let BridgeInstance;
+    let CentrifugeAssetInstance;
+
+    let resourceID;
+    let depositFunctionSignature;
+    let GenericHandlerInstance;
+    let depositData;
+    let proposal;
+
+    beforeEach(async () => {
+        await Promise.all([
+            BridgeInstance = await Helpers.deployBridge(destinationDomainID, accounts[0]),
+            CentrifugeAssetContract.new().then(instance => CentrifugeAssetInstance = instance)
```

Comment:
Pretty sure that is not how it should work as the first param should be metadata depositor.